packages/react/src/modules/guide/providers/KnockGuideProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import {
+  KnockGuideProvider as KnockGuideProviderCore,
+  type KnockGuideProviderProps,
+} from "@knocklabs/react-core";
+import React from "react";
+
+import { GuideDevTools } from "../components";
+
+// Re-export the core KnockGuideProvider, so we can add React specific functionality
```

Comment:
I decided to add the devtools component here and re-export the KnockGuideProvider so that we can use Telegraph without impacting non-web React usage. This means we need to export some additional things like `KnockGuideProviderProps` from react-core.